roles/smartgateway/templates/ceilometer-metrics-configmap.yaml.j2

```
@@ -0,0 +1,24 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ meta.name }}-smartgateway
+  namespace: {{ meta.namespace }}
+data:
+  "smartgateway_config.json": |
+    {
+      "AMQP1Connections": [
+        {
+          "URL": "{{ amqp_url | default('messaging-internal-' + meta.name + '.' + meta.namespace + '.svc:5672/collectd/telemetry') }}",
+          "DataSource": "{{ amqp_data_source | default('collectd') }}"
```

The logic in main.yml currently guarantees this, but for clarity I think...
```suggestion
          "DataSource": "{{ amqp_data_source | default('ceilometer') }}"
```